src/chemics/chemical_equation.py

```diff
@@ -39,13 +39,11 @@ class ChemicalEquation:
         Total number of moles on product side of the equation.
     prod_mass : float
         Total mass of the products.
-    balance : bool
-        Balance between atomic elements in reactants and products.
 
     Examples
     --------
     >>> ce = cm.ChemicalEquation('2 HCl + 2 Na -> 2 NaCl + H2')
-    >>> ce.balance
+    >>> ce.is_balanced
```

Comment:
This should be `>>> ce.is_balanced()` otherwise the example in the docstring will fail.

Reply:
Fixed.